api/swagger.yaml

```diff
@@ -487,6 +489,14 @@ definitions:
                 type: "string"
             example:
               [["noexec"]]
+      APISocketOptions:
+        description: "Optional configuration for the `apisocket` type."
+        type: object
+        required: ["isolation"]
+        properties:
+          isolation:
+            type: string
+            enum: ["none"]
```

Comment:
In future if we have other access options, this is how the Go representation of this look:

```go
type APISocketOptions struct {
    Access string `json:"access"`
    
    // Possibly in future, if we need parameters of other "access" modes:
    PolicyOptions struct {
       PolicyContent string `json:"path"`
       Endpoints []string `json:"endpoints"`
    }  `json:"access-policy"`
    SomeOtherAccessTypeOptions struct{…}
}
```